fink_filters/filter_kn_candidates/filter.py

```diff
@@ -2,13 +2,17 @@
 from pyspark.sql.types import BooleanType
 
 import pandas as pd
+import requests
+import os
 
 @pandas_udf(BooleanType(), PandasUDFType.SCALAR)
-def kn_candidates(knscore, drb, classtar, jd, jdstarthist, ndethist, cdsxmatch) -> pd.Series:
+def kn_candidates(objectId, knscore, drb, classtar, jd, jdstarthist, ndethist, cdsxmatch) -> pd.Series:
     """ Return alerts considered as KN candidates
```

Comment:
Could you update the docstring to specify that alerts are also sent via Slack if the environment variable `KNWEBHOOK` is defined?